packages/auth/src/app.ts

```diff
@@ -322,9 +322,7 @@ export class App {
           maxAge: 60 * 1000,
           signed: true
         },
-        // Only accepts Middleware<DefaultState, DefaultContext> for some reason, this.koa is Middleware<DefaultState, AppContext>
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        koa as any
```

Comment:
Just didnt need to do this anymore... this might have been where it was fixed https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65100